### PR TITLE
[JENKINS-46669] Reword SetupWizard Continue as admin button

### PR DESCRIPTION
**Pull request by samhrncir**
See [JENKINS-46669](https://issues.jenkins-ci.org/browse/JENKINS-46669).

### Proposed changelog entries

* Entry 1: Reworded "Continue as admin" button of the plugin setup wizard.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@fqueiruga
@timja

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There are at least 2 approvals for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [x] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by fqueiruga — APPROVED** [review]

**Comment by fqueiruga** [comment]
I don't see any reason to not approve it. @timja are you aware if there's a copyediting team that should review this?

**Comment by timja** [comment]
> I don't see any reason to not approve it. @timja are you aware if there's a copyediting team that should review this?

yes but it's only on jenkins-infra so can't tag them here

File: core/src/main/resources/jenkins/install/pluginSetupWizard.properties
Comment (daniel-beck) [inline core/src/main/resources/jenkins/install/pluginSetupWizard.properties:68]: Unsure, but perhaps?

```suggestion
installWizard_skipFirstUser=Skip and continue as admin
```

Comment (samhrncir) [reply core/src/main/resources/jenkins/install/pluginSetupWizard.properties:68]: I agree with the need to trim it.

The other option to be more concise could be simply _skip_ as there is clarification at the end of the setup wizard:
postUserCreateFormClickedContinueAsAdmin

**Review by timja — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
+100, this button has been always confusing for me

**Review by rsandell — APPROVED** [review]

**Review by alecharp — APPROVED** [review]
This is great, but I think we need to do the same on translations.

**Comment by fqueiruga** [comment]
Pinging @LinuxSuRen in case you want to revisit the cn translation.

**Comment by LinuxSuRen** [comment]
@fqueiruga Thanks for reminding me. This is a good improvement.

**Review by MarkEWaite — APPROVED** [review]

**Comment by res0nance** [comment]
Merging in 24h if no negative feedback

**Pull request merged**
